src/main/java/com/artipie/misc/Property.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2021 artipie.com
+ * https://github.com/artipie/artipie/LICENSE.txt
+ */
+package com.artipie.misc;
+
+import com.artipie.ArtipieException;
+import java.util.Optional;
+
+/**
+ * Obtains value of property from properties which were already set in
+ * the environment or in the file.
+ * @since 0.23
+ */
+public final class Property {
+    /**
+     * Name of the property.
+     */
+    private final String name;
+
+    /**
+     * Ctor.
+     * @param name Name of the property.
+     */
+    public Property(final String name) {
+        this.name = name;
+    }
+
+    /**
+     * Obtains long value of the property set properties or file
+     * with values of the properties.
+     * @param defval Default value for property
+     * @return Long value of or default value.
+     * @throws ArtipieException In case of problem with parsing value of the property
+     */
+    public long asLongOrDefault(final String defval) {
```

Comment:
I'd suggest to change type of the parameter from String to long, as this method is designed to obtain and return longs